.changie.yaml

```diff
@@ -77,15 +78,22 @@ custom:
 
 footerFormat: |
   {{- $contributorDict := dict }}
-  {{- /* any names added to this list should be all lowercase for later matching purposes */}}
-  {{- $core_team := list "michelleark" "peterallenwebb" "emmyoop" "nathaniel-may" "gshank" "leahwicz" "chenyulinx" "stu-k" "iknox-fa" "versusfacit" "mcknight-42" "jtcohen6" "aranke" "dependabot[bot]" "snyk-bot" "colin-rogers-dbt" }}
+  {{- /* ensure all names in this list are all lowercase for later matching purposes */}}
+  {{- $exclude_authors := list "dependabot[bot]" "snyk-bot"}}
+  {{- $core_team := splitList " " .Env.CORE_TEAM }}
```

Comment:
Does `Env.CORE_TEAM` point directly to the GH team? In other words, granting access to the team carries through to this workflow? Pretty great.

Reply:
Sadly no.  But that would be great.  I have to grab the team in another workflow that we use to generate the changelogs on release and set it as an environment variable.  See https://github.com/dbt-labs/dbt-release/pull/60/files

Reply:
lol, oh well, one can dream